app/pages/Logs.js

```diff
@@ -53,17 +52,18 @@ class Logs extends React.Component {
             <Feed.Event>
               <Feed.Content>
                 <Feed.Summary>
-                  <Label size="mini" color={this.labelColor(entry.type)}>
-                    {capitalize(entry.type)}
+                  <Label color={determineLabelColor(entry.type)} image horizontal>
+                    <Icon name={determineLabelIcon(entry.type)} />
```

Comment:
You do use Icon here, but did not define it at the top.

Reply:
Whoops, should be fixed with d538230